src/game/server/tf/tf_player.cpp

```diff
@@ -1432,6 +1433,26 @@ void CTFPlayer::GiveDefaultItems()
 	else if ( !tf2v_random_weapons.GetBool() )
 		ManageRegularWeapons( pData );
 
+	if (tf_halloween.GetBool())
+	{
+		int iItemID = 5617;
+		CEconItemDefinition *pItemDef = GetItemSchema()->GetItemDefinition(iItemID);
+		if (!pItemDef)
+			return;
```

Comment:
Shouldn't break out here, even if it would technically be an error that there isn't a item definition